bazel/repositories.bzl

```diff
@@ -592,7 +592,12 @@ def _com_google_absl():
     )
 
 def _com_google_protobuf():
-    external_http_archive("rules_python")
+    external_http_archive(
+        name = "rules_python",
+        patches = ["@envoy//bazel:rules_python.patch"],
```

Comment:
Please add a comment on why the patch is needed and when we can revert with a TODO.